_data/banking.yml

```diff
@@ -381,6 +381,13 @@ websites:
       hardware: Yes
       doc: http://www.intesasanpaolo.com/servizi-multicanale/sicurezza.jsp
 
+    - name: La Banque Postale
+      url: https://www.labanquepostale.fr/
+      twitter: labanquepostale
```

Comment:
Please use the same case that their Twitter handle uses: [`LaBanquePostale`](https://twitter.com/labanquepostale).